proposals/0002-ledger-car.md

```diff
@@ -0,0 +1,277 @@
+---
+simd: '0002'
+title: Ledger-CAR file format
+authors:
+  - Richard Patel (Jump Crypto)
+  - Peter Rabbitson (Protocol Labs)
+  - Jonathan Victor (Protocol Labs)
+category: Interface
+type: Standards Track
+status: Draft
+created: 2022-11-02
+---
+
+## Summary
+
+Ledger-CAR is a streaming file format for Solana ledger data.
+
+Ledger data broadly contains all block information required to validate all state transitions between two slots.
+
+Specifically, this includes two types of data:
+* Proof-of-History parameters
+* Transactions (both user txs and consensus txs)
+
+To support distribution over public networks, Ledger-CAR files are generated deterministically and are stored data via content-addressable IPLD objects.
+
+## Specification
+
+### Solana Entries
+
+The canonical representation of block data is the "entries" format.
+Each entry contains proof-of-history tick parameters and a subslice of transactions recorded in the block.
+
+Entries are serialized as Bincode little-endian encoding.
+
+```python
+class Entry:
+  # PoH parameters
+  num_hashes: int
+  prev_hash: Hash
+  # Block data
+  transactions: list[Tx]
+
+class Block:
+  slot: int
+  entries: list[Entry]
+```
+
+### CARv1 File Format
+
+The **Content-addressable ARchive** is a streaming container format that serves as the basis of the Ledger-CAR file format.
+[CARv1] files store a number of binary blobs of arbitrary content and size.
```

Comment:
Protocol Labs uses the term (IPLD) "block" instead of blobs. However, "blob" was chosen here to resolve the naming conflict with Solana block. Should add a section explaining the newly introduced terms